__init__.py

```diff
@@ -240,6 +205,70 @@ def ctrl_chg_day(bs):
     else:
         updated = False
 
+def ctrl_chg_thm(bs):
+    global MODE, updated
+    updated = True
+    global conf
+    global theme
```

Comment:
`global conf` and `global theme` are probably unnecessary here.

You only need to specify them as global if you write to the variable (specifically, expressions like `conf = foo`).

Reply:
actually, it is written to `theme` here, but `global conf` should be really unneccessary